lib/orbital/models/response.rb

```diff
@@ -80,6 +80,27 @@ def self.orbital_response_params(response)
         }
       end
 
+      def cancel
+        begin
+          error_details = JSON.parse(message)
+          original_message = nil
+        rescue
+          error_details = {}
+          original_message = message
+        end
+        error_details['original_message'] = original_message unless original_message.blank?
+        error_details['payment_plugin_status'] = 'CANCELED'
+
+        updated_attributes = {
+            :message => error_details.to_json,
+            :success => false,
+            :updated_at => Time.now.utc
```

Comment:
Tried to do so; how can I get `@clock` in `response.rb`?

Looks like we use `Time.now.utc` in other plugins as well, e.g, CyberSource and PayPal